cdap-ui/graphql/resolvers/applicationResolvers.js

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+const merge = require('lodash/merge'),
+  urlHelper = require('../../server/url-helper'),
+  cdapConfigurator = require('../../cdap-config.js'),
+  resolversCommon = require('./resolvers-common.js');
+
+let cdapConfig;
+cdapConfigurator.getCDAPConfig()
+  .then(function (value) {
+    cdapConfig = value;
+  });
+
+const applicationsResolver = {
+  Query: {
+    applications: async (parent, args, context, info) => {
+      const namespace = args.namespace
+      const options = resolversCommon.getGETRequestOptions();
+      options['url'] = urlHelper.constructUrl(cdapConfig, `/v3/namespaces/${namespace}/apps`);
```

Comment:
nit: Don't use `[` notation. Can't it be just `options.url = ...`? I think we can change it across all resolvers.

Reply:
I get `url is not defined` when running a query

Reply:
Options is something that we define locally. I don't believe we should see any when running queries. Can you talk a little bit more about the problem you are seeing here?

Reply:
I wrote ` options.url = urlHelper.constructUrl(cdapConfig, `/v3/namespaces/${namespace}/apps`);` and got the error above. The `options` object is built in the `resolversCommon.getGETRequestOptions()` function.